workflow/scripts/plot-primer-clipping.py

```diff
@@ -0,0 +1,151 @@
+import pandas as pd
+import pysam
+from intervaltree import Interval, IntervalTree
+import altair as alt
+
+# read primer bedpe to df
+PRIMER = pd.read_csv("resources/primer.bedpe", delimiter="\t", header=None)
```

Comment:
We could include this path via the config.yaml. This would provide more flexibility down the road.

Reply:
This was somehow double posted. This closing this.